lib/common/client/middleware/response/raise_error.rb

```diff
@@ -5,7 +5,7 @@ module Middleware
       module Response
         class RaiseError < Faraday::Response::Middleware
           def on_complete(env)
-            return if env.success?
+            return if env.status < 500
```

Comment:
I need to double check on exceptions returned from MHV. some of the that are returned as 500 by MHV or possibly even in the madeup range of 900 level http status codes might also need to be handled by ClientResponse and not by breakers.